src/features/pagination/Pagination.scss

```diff
@@ -6,10 +6,6 @@
     .bdl-Pagination-count {
         display: flex;
         align-items: center;
-
-        .be-is-small & {
-            display: none;
-        }
```

Comment:
I verified this component is only used for Explorer and Picker. Moving this style to the ContentPicker stylesheet so it's only applied for that element